packages/docs/docs/docs/deploying.md

```diff
@@ -127,27 +133,27 @@ To start working with it, open a separate terminal and run:
 ganache-cli --port 9545 --deterministic
 ```
 
-Once we have done that, let's go back to the original terminal and 
+Once we have done that, let's go back to the original terminal and
 run the following command:
 
 ```console
-zos session --network local --from 0x1df62f291b2e969fb0849d99d9ce41e2f137006e --expires 3600 
-``` 
+zos session --network local --from 0x1df62f291b2e969fb0849d99d9ce41e2f137006e --expires 3600
+```
 
 The `session` command starts a session to work with a desired network.
-In this case, we are telling it to work with the `local` network with the 
-`--network` option, and also setting a default sender address for the 
-transactions we will run with the `--from` option. Additionally, the 
-`expires` flag allows us to indicate the session expiration time in seconds. 
-
-> Note that we are using a specific address for the `--from` option which 
-is different to the default address that `ganache-cli` would use. 
-This is because we need to use different addresses in order to create 
-upgradeable contracts and to query them. This problem is know as the  
-"transparent proxy issue" and you can read more about it in the 
-[ZeppelinOS upgrades pattern section](pattern.md). 
-
-Now that everything has been setup, we are ready to deploy the project. 
+In this case, we are telling it to work with the `local` network with the
+`--network` option, and also setting a default sender address for the
+transactions we will run with the `--from` option. Additionally, the
+`expires` flag allows us to indicate the session expiration time in seconds.
+
+> Note that we are using a specific address for the `--from` option which
+is different to the default address that `ganache-cli` would use.
+This is because we need to use different addresses in order to create
+upgradeable contracts and to query them. This problem is
+known as the "transparent proxy issue" and you can read more about it in the
```

Comment:
Should we rephrase it to avoid using "issue", since we're at it?

Reply:
sure thing